### ISSUE TITLE
Local development isn't working

### ISSUE THREAD

**Issue by marlonmarcello** [bigcommerce/checkout-sdk-js-example#22]
While developing locally I get a bunch of 404s:
```
http://localhost:8080/api/storefront/checkout-settings
http://localhost:8080/internalapi/v1/store/countries
http://localhost:8080/internalapi/v1/shipping/countries
```
And it makes sense, how does the library knows which store to use? Where do I set the store settings? Like hash, token and secret?

**Comment by RobinOng** [comment]
Hey @marlonmarcello,

As far as I know, this example does not contain a development server. What setup are you using?

**Comment by marlonmarcello** [comment]
I added an `index.html` file to `dist/` and fired up a simple http server with python, like so `python -m SimpleHTTPServer 8080`

**Comment by marlonmarcello** [comment]
I added the required `<div id="checkout-app"></div>` and the `<script>` references the compiled JS from running `npm run dev`

**Comment by marlonmarcello** [comment]
Is there a better way to test this locally?

**Comment by RobinOng** [comment]
Hi, here is some documentation to get you started: https://developer.bigcommerce.com/stencil-docs/customizing-checkout/checkout-sdk-quickstart. Don't hesitate to reach out to us if you have any question.

**Comment by marlonmarcello** [comment]
@RobinOng so this SD only works inside Stencil? Can I use it as stand alone?

**Comment by RobinOng** [comment]
Yes, it only works inside Stencil. You cannot use it as a standalone.
